go-controller/pkg/ovn/multicast.go

```diff
@@ -76,8 +85,27 @@ func getMulticastACLEgrMatch() string {
 	return getACLMatchAF(ipv4Match, ipv6Match)
 }
 
-func getMcastACLName(nsORpg, mcastSuffix string) string {
-	return joinACLName(nsORpg, mcastSuffix)
+func getMcastACLDbIDs(ns string, actionID mcastActionID, aclT aclType, controller string) *libovsdbops.DbObjectIDs {
+	// there 4 possible IDs for global default multicast acls deny/allow+ingress/egress
+	// and 2 types for namespaced-acls allow+ingress/egress
+	internalID := string(actionID) + "_" + aclTypeToPolicyTypeStr(aclT)
+	if ns == "" {
+		// global multicast ACLs
+		return libovsdbops.NewDbObjectIDs(libovsdbops.ACLMulticastGlobal, controller,
+			map[libovsdbops.ExternalIDKey]string{
+				libovsdbops.TypeKey: internalID,
+			})
+	} else {
```

Comment:
I can't , because owner types are different `libovsdbops.ACLMulticastGlobal` and `libovsdbops.ACLMulticastNamespace`, that is why I just left 2 branches